ledger-core-tezos/src/tezos/transactions/TezosLikeTransaction.cpp

```diff
@@ -153,6 +171,19 @@ namespace ledger {
                 writer.writeByte(static_cast<uint8_t>(api::TezosOperationTag::OPERATION_TAG_GENERIC));
             }
 
+            // If tx was forged then nothing to do
+            if (!_rawTx.empty()) {
+                writer.writeByteArray(_rawTx);
```

Comment:
I think you are missing a fix here : https://github.com/LedgerHQ/lib-ledger-core/commit/672428c5920047d0d20f3fe418f77b2ace46b110